jekyll/_cci2_ja/pipeline-variables.md

```diff
@@ -1,44 +1,30 @@
-| pipeline.git.tag           | パイプラインをトリガーするためにプッシュされた git タグの名前。 タグでトリガーされたパイプラインでない場合は、文字列は空です。                                                                                                                                                                                                                             |
-| pipeline.git.branch        | パイプラインをトリガーするためにプッシュされた git タグの名前。                                                                                                                                                                                                                                                             |
-| pipeline.git.revision      | 現在ビルドしている長い git SHA（４０文字）                                                                                                                                                                                                                                                                      |
-| pipeline.git.base_revision | 現在ビルドしているものより前のビルドの長い git SHA （４０文字） <br/><br/> **Note:** While in most cases `pipeline.git.base_revision` will be the SHA of the pipeline that ran before your currently running pipeline, there are some caveats. ブランチの最初のビルドの場合、変数は表示されません。 また、ビルドが API からトリガーされた場合も変数は表示されません。 |
-| pipeline.in_setup          | True if the pipeline is in the setup phase, i.e. running a [setup workflow]({{ site.baseurl }}/2.0/dynamic-config/).                                                                                                                                                                           |
-| pipeline.trigger_source    | The source that triggers the pipeline, current values are `webhook`, `api`, `scheduled_pipeline`                                                                                                                                                                                               |
-| pipeline.schedule.name     | The name of the schedule if it is a scheduled pipeline. Value will be empty string if the pipeline is triggerd by other sources                                                                                                                                                                |
-| pipeline.schedule.id       | The unique id of the schedule if it is a scheduled pipeline. Value will be empty string if the pipeline is triggerd by other sources                                                                                                                                                           |
-{: class="table table-striped"}
-
-例えば下記のようにします。
+パイプライン値は、あらゆるパイプライン構成で使用可能であり、事前に宣言することなく使用できます。
+
+For a full list of values and built-in environment variables, see the [Project Values and Variables guide]({{site.baseurl}}/2.0/variables/#pipeline-values).
+
+{% include snippets/pipeline-values.md %}
+
+Usage example:
```

Comment:
[question; confirmation required]

same here... the localization to 🇯🇵 seems missing; hmm perhaps this is intended?

Reply:
[edit]
this will be tackled in later PRs